precompile/auth/IAuth.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+address constant AUTH_PRECOMPILE_ADDRESS = 0x1000000000000000000000000000000000000005;
+
+IAuth constant AUTH_CONTRACT = IAuth(
+    AUTH_PRECOMPILE_ADDRESS
+);
+
+interface IAuth {
+    function account(address evmAddress) external view returns (bytes calldata addr);
```

Comment:
I think the return type should be string

Reply:
[f7b9f5b](https://github.com/xpladev/xpla/pull/161/commits/f7b9f5bb747af3954ded13249eb1dc3796b79674)